src/app/demo/default/default.component.ts

```diff
@@ -43,10 +48,12 @@ export type ChartOptions1 = {
 
```

Comment:
ChartOptions1 is not needed. it can be merge in to ChartOptions and used everywhere.

Reply:
done